crates/project/src/search.rs

```diff
@@ -327,6 +327,10 @@ impl SearchQuery {
         matches
     }
 
+    pub fn is_empty(&self) -> bool {
```

Comment:
Would it make sense to trim it?

Reply:
Yeah, I think spaces/tabs is fine, so this is just empty-empty